src/www/views/servers_view/server_list_item/server_card/index.ts

```diff
@@ -95,6 +95,7 @@ const sharedCSS = css`
     color: var(--outline-text-color);
     font-size: var(--server-name-size);
     margin-bottom: var(--outline-mini-gutter);
+    word-break: break-all;
```

Comment:
```suggestion
    /* To break the line in case a sequence of word characters is longer than the line.
       See https://github.com/Jigsaw-Code/outline-client/issues/1372. */
    word-break: break-all;
```

Reply:
I'm taking my first steps in open source, I'll be glad for any advice. Is it always important to leave such comments if or under what conditions?

Reply:
He makes a good point, @fortuna, should we be commenting all instances of `work-break: break-all;`? [There's another one here.](https://github.com/Jigsaw-Code/outline-client/blob/master/src/www/views/servers_view/server_list_item/server_card/index.ts#L105)